src/c++/library/grpc_client.cc

```diff
@@ -1091,9 +1091,17 @@ InferenceServerGrpcClient::PreRunProcessing(
   infer_request_.set_id(options.request_id_);
 
   infer_request_.mutable_parameters()->clear();
-  if (options.sequence_id_ != 0) {
```

Comment:
This is an example of the type of code that needs to continue to work for backwards compatibility. Not sure if that is possible with current design but may be...

Reply:
Are people going to be writing their own grpc_client.cc? I assumed that it should be enough to be able to do both
```
...
const uint64_t sequence_id  = 1000;
tc::InferOptions options(model_name);
options.sequence_id_ = sequence_id;
...
```
as well as 
```
...
const std::string sequence_id  = "1000";
tc::InferOptions options(model_name);
options.sequence_id_ = sequence_id;
...
```